.bazelrc

```diff
@@ -44,6 +44,20 @@ build:release --workspace_status_command="node ./tools/bazel-stamp-vars.js"
 # does not generate factory files which are needed for AOT.
 build --define=compile=legacy
 
+#######################
+# Remote HTTP Caching #
+#######################
+build --remote_http_cache=https://storage.googleapis.com/angular-team-cache
+build --remote_accept_cached=true
+build --remote_upload_local_results=false
+
+######################################
+# Remote HTTP Caching writes support #
+# Turn on these settings with        #
+#  --config=-http-caching            #
+######################################
+build:remote-http-caching --remote_upload_local_results=true
```

Comment:
This only works for authorized users right?

Reply:
Correct

Reply:
Sounds good! 💯 will there be a doc or some guidelines on how one can authenticate?